### PR TITLE
🎉  wizard: add metadata-export, love to chart revision

### PR DESCRIPTION
**Pull request by lucasrodes**
### TL;DR
No need to review the code in detail, just try out the tool: 

```
etl-wizard charts
```

and in see how it feels. Most changes should not be noticeable.

You can play with the GPT tool to tweak the FASTT. You can find it in the last step, where you have a preview of the charts (after submitting the variables).
 
### Summary of what has been done
- Add **Metadata Export** to Wizard.
  - Added under new tab "Actions" in `Dataset Explorer`
  - Added in the guidelines after submitting Garden create step.
- Give some love to our **Chart Revision Baker** tool
  - Simplify and document some parts of the python back-end code
  - Better management of the streamlit `st.session_state` variable
  - Add GPT-based FASTT tweaking options to the chart preview section.
- Add "Next step" button in the guidelines after submitting ETL create step.
- Add harmonize in data explorer.

### CODE REVIEW

**Comment by lucasrodes** [comment]
@pabloarosado no need to review the code in detail. Can just run the chart revision tool and see how it feels

```
etl-wizard charts
```

Optionally, you can set DEBUG=True in the `apps/wizard/charts/search_config.py` file, to pre-select a dataset mapping.

**Comment by pabloarosado** [comment]
Hey Lucas, everything looks great!
A minor thing I noticed, when submitting chart revisions, I get the following message: "Entered last step, but submission is None?".
Screenshot 2024-02-16 at 11 03 30
I don't know what it means, the submission seems to have worked well (they are in my local queue). And on that topic, I think it would be good to clarify that clicking on the "Finish" button means submitting chart revisions. Maybe instead of "Finish" you could call it "Submit chart revisions", or otherwise add a text above the button saying that.

Apart from that, I haven't looked into the code, but I understand you haven't changed anything affecting the data/metadata (except for the GPT suggestions, which seem to work as expected), so you can just merge so people can start using the tool. Good work!

**Review by pabloarosado — APPROVED** [review]
The wizard works well.
